vendor-overwrites/colorpicker/colorconverter.js

```diff
@@ -74,16 +107,21 @@ const colorConverter = (() => {
     const first = num[0];
     if (/rgb/i.test(type)) {
       const k = first.endsWith('%') ? 2.55 : 1;
-      const [r, g, b] = num.map(s => parseFloat(s) * k);
-      return {type: 'rgb', r, g, b, a};
+      const [r, g, b] = num.map(s => {
```

Comment:
There should be no need to change the parsing process. Just add the validation after L109:
```js
const valid = first.endsWith("%") ? validatePercentage : validateNum;
if (num.slice(0, 3).some(s => !valid(s))) {
  return null;
}
```

Reply:
The parsing wasn't optimal.

If `rgb(12a, 255g, 10adv)` was entered, it would not show an error, but would parse to `rgb(12, 255, 10)` correctly. With this change, the entered value would show an error.

Reply:
I don't understand. Do you mean the parser should not return `null` while parsing `rgb(12a, 255g, 10adv)`?

Reply:
It's just easier to deal with `-1` than `NaN`  (the result of parsing `null`)